src/RA_Dlg_AchEditor.cpp

```diff
@@ -398,8 +396,8 @@ BOOL CreateIPE(int nItem, CondSubItems nSubItem)
 
     HWND hList = GetDlgItem(g_AchievementEditorDialog.GetHWND(), IDC_RA_LBX_CONDITIONS);
 
-    RECT rcSubItem;
-    ListView_GetSubItemRect(hList, nItem, ra::etoi(nSubItem), LVIR_BOUNDS, &rcSubItem);
+    RECT rcSubItem{};
+    GSL_SUPPRESS_ES47 ListView_GetSubItemRect(hList, nItem, ra::etoi(nSubItem), LVIR_BOUNDS, &rcSubItem);
```

Comment:
This was the macro that needed the use `nullptr` rule suppressed. Our input is fine.